Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    hocon (1.0.0)
+    hocon (1.0.1)
```

Comment:
tbh I have no idea what the convention is for committing the `.lock` files to VCS, because I don't have much experience managing ruby gems.

It seems to me that it would be a good practice to commit them because it allows for repeatable builds, which seems highly desirable.  However, if it's considered an anti-pattern to commit them I'd be OK with removing it I guess.  Would like to understand the rationale because it just seems like it's going to make the build non-deterministic, but... :dancers: